File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        c = 0
        for word in words:
            f = 1
            for i in word:
                if i not in allowed:
                    f = 0
            if f == 1:
                c+=1
                
        return c

File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        int c = 0;
        vector<int> setA(26, 0);
        for(char c:allowed){
            setA[c-'a'] = 1;
        }
        for(string& s: words){
            int f = 1;
            for(char c: s){
                if(setA[c-'a']!=1){
                    f = 0;
                }
            }
            if(f==1){
                c++;
            }
        }
        return c;
    }
};